docs/dev/dev-proxy/how-to/create-custom-plugin.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Create a custom plugin
+description: How to create a custom plugin for Dev Proxy
+author: eliostruyf
+ms.author: 
+ms.date: 03/28/2024
+---
+
+# Create a custom plugin
+
+In this article, you will learn how to create a custom plugin for the Dev Proxy. By creating plugins for Dev Proxy, you can extend its functionality and add custom features to fit your needs.
+
+## Prerequisites
+
+Before you start creating a custom plugin, make sure you have the following prerequisites:
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
+- [Visual Studio Code](https://code.visualstudio.com/)
```

Comment:
You don't need to use VSCode. We could say we recommend it, but I wouldn't say it's a requirement.

Reply:
Removed the VSCode reference